api/configuration/health.ts

```diff
@@ -0,0 +1,20 @@
+import {EnvironmentConfigServices} from '../interfaces/environment.config'
+import {getConfigValue} from './index'
+import { SERVICES_CCD_COMPONENT_API_PATH } from './references'
+import { SERVICES_CCD_DATA_STORE_API_PATH } from './references'
+import { SERVICES_IDAM_LOGIN_URL } from './references'
+import { SERVICE_S2S_PATH } from './references'
+import { SERVICES_TERMS_AND_CONDITIONS_URL } from './references'
+
+export const healthEndpoints = (): EnvironmentConfigServices => {
+
+  const HEALTH = '/health'
+
+  return {
+    ccdComponentApi: getConfigValue(SERVICES_CCD_COMPONENT_API_PATH) + HEALTH,
```

Comment:
String literals for all of these...?

Reply:
i know i know. was just trying to match what was in MO, since the objective was to match it like for like.

Reply:
i have made the change though